docs/user-guide.md

```diff
@@ -5,9 +5,9 @@ footage from those cameras. It includes instructions on K8s cluster setup. If yo
 cluster of Raspberry Pi 4's, see the [Raspberry Pi 4 demo](./end-to-end-demo-rpi4.md).
 
 ## Getting Started
-To get started using Akri, you must first decide what you want to discover and whether Akri currently supports a protocol
-that can be used to discover resources of that type. To see the list of currently supported protocols, see our
-[roadmap](./roadmap.md).
+To get started using Akri, you must first decide what you want to discover and whether Akri currently supports a Discovery Handler
+that can be used to discover resources of that type. Akri discovers devices via Discovery Handlers, which are often protocol implementations that understand filter information passed via an Akri Configuration. To see the list of currently supported Discovery Handlers, see our
+[roadmap](./roadmap.md). These Discovery Handlers can be embedded in the Akri Agent (via the Helm setting `agent.full=true`), which runs on every worker node in a cluster, or run in their own Pods (default).  
```

Comment:
nit: maybe we should state the default first and then add that this can be changed (with a a link to docs showing how)

Reply:
Are you suggesting a separate page of docs to describe how to request embedded DHs?  To me changing this is just setting `agent.full=true` but maybe elaboration makes sense or i can see linking to a section within the doc about it

Reply:
Re-reading it, it definitely makes sense to not mention the two DH strategies here. I added it to one of the helm charts notes and it is still in the flow section as a note. You are right that we should focus on emphasizing the default

Reply:
changed in afd6d976f8124bc7